src/sparseml/pytorch/optim/quantization/helpers.py

```diff
@@ -37,15 +40,27 @@
 ]
 
 
+_QUANTIZABLE_MODULE_TYPES = (
+    {
+        module
+        for name, module in getmembers(nni.modules.fused, isclass)
+        if "Conv" in name or "Lienar" in name
```

Comment:
Also, this seems somewhat fragile. Is there any reason we can list out the module types or the base types?

Reply:
no, we can list them, only reason I didn't was because there are 12, will update